x-pack/legacy/plugins/siem/cypress/integration/lib/util/helpers.ts

```diff
@@ -25,3 +25,25 @@ export const loginAndWaitForPage = (url: string) => {
 
 export const waitForTableLoad = (dataTestSubj: string) =>
   cy.get(dataTestSubj, { timeout: DEFAULT_TIMEOUT });
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const cookiesCache: Record<string, Record<string, any>> = {};
+
+export const saveCookies = () => {
```

Comment:
Alternatively we can whitelist the cookie by adding the following code to `x-pack/legacy/plugins/siem/cypress/support/index.js`:

```
Cypress.Cookies.defaults({
  whitelist: 'sid',
});
```

The advantage of this alternative approach is we don't have to explicitly save / load cookies in tests. I re-ran the tests ~ 12 times after making the above change (and commenting-out the explicit load / save calls), and the tests passed every time.

Would you be willing to verify this works in your environment?

Reply:
Thanks for the suggestion I'll take a look. I tried to use at first `Cyoress.Cookies.preserveOnce()` and was not working for me. Meanwhile I was reading about similar issues I saw that Whitelist was not working as well, this is the main reason why I didn't try it. I'll take a look and if works I'll use it since that is going to help us to save lines of code :)